### PR TITLE
Applied needed styling for form-fields

### PR DESCRIPTION
**Pull request by 50Hannah000**

### CODE REVIEW

File: addon/styles/components/form-field.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
 .form-field {
   display: inline;
+
+  div {
+    width: 70%;
+    display: inline-block;
+  }
```
Comment (michaelw8) [inline addon/styles/components/form-field.scss]: Why did you add this? I think it doesn't belong here. It's something you just can add in de resource-manager, isn't it?

**Comment by cbroeren** [comment]
When you want this styling, please add it in the project you're using it
